analyses/cms-open-data-ttbar/analysis.py

```diff
@@ -138,71 +138,48 @@ def define_trijet_mass(df: ROOT.RDataFrame) -> ROOT.RDataFrame:
         """
         ROOT::VecOps::Construct<ROOT::Math::PxPyPzMVector>(
             ROOT::VecOps::Construct<ROOT::Math::PtEtaPhiMVector>(
-                Jet_pt[Jet_pt_mask], Jet_eta[Jet_pt_mask], Jet_phi[Jet_pt_mask], Jet_mass[Jet_pt_mask]
+                Jet_pt_masked, Jet_eta[Jet_pt_mask], Jet_phi[Jet_pt_mask], Jet_mass[Jet_pt_mask]
             )
         )
         """,
         )
     )
 
     # Build trijet combinations
-    df = df.Define("Trijet", "ROOT::VecOps::Combinations(Jet_pt[Jet_pt_mask],3)")
-    df = df.Define("nTrijet", "Trijet[0].size()")
+    df = df.Define("Trijet", "Combinations(Jet_pt_masked, 3)")
```

Comment:
I just realized naming of this column is "wrong": this column does not contain a "trijet", it contains indices that to make trijets. what's a better name? "trijet_idx" maybe?

Reply:
Thanks, it is slightly confusing. I've added `_idx` to the `Trijet`